e2e/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "e2e",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test:e2e": "npx playwright test ",
+    "test:e2e:ui": "npx playwright test --ui"
```

Comment:
delete? this one doesn't seem used

Reply:
I was using the ui all the time when developing playwright tests for the grants search - it lets you see where it's clicking on the page and such - but open to discussion if you feel strongly

![image](https://github.com/navapbc/platform-test-nextjs/assets/93001277/cfea17ef-8966-44a0-8217-a0915ddff366)

Reply:
oh i see, sounds good.